rfc/text/advanced/ap_50_payload_passthru_mode.md

```diff
@@ -118,17 +118,18 @@ as well as `ERROR` messages in the case of failures.
 
 **ppt_scheme Attribute**
 
-The `ppt_scheme` identifies the Key Management Schema. It is a required string attribute. This attribute can contain 
-the name or identifier of a key management provider that is known to the target peer, so it can be used to obtain 
-information about encryption keys. A *Router* can recognize that `Payload Passthru Mode` is in use
-by checking the existence and non-empty value of this attribute within the options of `CALL`, `PUBLISH` and `YIELD` 
-messages.
+The `ppt_scheme` identifies the Payload Schema. It is a required string attribute. For End-2-End Encryption flow
+this attribute can contain the name or identifier of a key management provider that is known to the target peer, 
+so it can be used to obtain information about encryption keys. For gateways and external schemas this can
+contain the name of related technology such as `mqtt`, `amqp`, `stomp`. A *Router* can recognize that 
```

Comment:
I would only refer to mqtt, not the other ones, and only a mapping for mqtt is defined, and not for the other ones (and there is no interest anyways I guess in those)

Reply:
Well, agree. Let's rephrase it to `mqtt` for now and add that other schemas may be added later.

Reply:
cool, I think this makes sense to add an AP section that describes "mqtt" (well, currently, there isn't even a proposal, only CB impl., but it is really quite straight forward - making use of the building blocks we already define in this PR)